que-locks.gemspec

```diff
@@ -21,10 +21,10 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", '~> 2.0'
-  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "bundler", "~> 2.0"
```

Comment:
The diff has some splash damage here, do you have an auto-linter setup or something?

Reply:
I'll fix this